### PR TITLE
openssl list: Fix help text about -cipher-algorithms option

### PR DESCRIPTION
**Pull request by faramir-dev**
Fixes openssl#19133

##### Checklist

- [x] documentation is added or updated
- [ ] tests are added or updated

### CODE REVIEW

**Review by paulidale — APPROVED** [review]

File: apps/list.c [hunk 1]
```diff
@@ -1476,7 +1476,7 @@ const OPTIONS list_options[] = {
     "List of cipher commands (deprecated)"},
 #endif
     {"cipher-algorithms", OPT_CIPHER_ALGORITHMS, '-',
-     "List of cipher algorithms"},
+     "List of symetric cipher algorithms"},
```
Comment (mattcaswell) [inline apps/list.c:1479]: typo: symmetric

File: doc/man1/openssl-list.pod.in [hunk 2]
```diff
@@ -129,10 +129,10 @@ This option is deprecated. Use B<cipher-algorithms> instead.
 Display a list of cipher commands, which are typically used as input
 to the L<openssl-enc(1)> or L<openssl-speed(1)> commands.
 
-=item B<-digest-algorithms>, B<-kdf-algorithms>, B<-mac-algorithms>,
-B<-cipher-algorithms>
+=item B<-cipher-algorithms>, B<-digest-algorithms>, B<-kdf-algorithms>,
+B<-mac-algorithms>,
 
-Display a list of cipher, digest, kdf and mac algorithms.
+Display a list of symetric cipher, digest, kdf and mac algorithms.
```
Comment (mattcaswell) [inline doc/man1/openssl-list.pod.in:135]: symmetric

**Review by mattcaswell — CHANGES_REQUESTED** [review]

**Comment by paulidale** [comment]
Still good.  I can't spell today :disappointed:

**Review by t8m — APPROVED** [review]

**Review by mattcaswell — APPROVED** [review]

**Comment by hlandau** [comment]
Merged to master and 3.0. Thank you.

**Pull request closed without merging**